definitions/divorce/json/FixedLists.json

```diff
@@ -581,12 +581,6 @@
     "ListElementCode": "feesHelpWith",
     "ListElement": "Help with fees"
   },
-  {
-    "LiveFrom": "01/01/2017",
```

Comment:
what about existing cases that already have "cheque"?

Reply:
I had that question too before I started 🤷‍♂  need to clarify if there are indeed no such cases

Reply:
do you know if we would be able to upload this on prod if there are any cheques?

Reply:
Just checked on prod, there are only 11 cases in total that use the cheque option currently, last one modified is today, the 6th Feb. Updating ticket with details and pinging Izzy

Reply:
ok, so we need to do data migration. Apparently @HarryH96 knows how.